posts/2019/05/spyder-4-coming-closer-beta2-is-now-available.md

```diff
@@ -0,0 +1,161 @@
+<!--
+.. title: Spyder 4.0 coming closer, beta2 is now available!
+.. slug: spyder-4-beta2-release
+.. date: 2019-05-16 16:38:50 UTC-05:00
+.. author: Gonzalo Peña-Castellanos
+.. tags: Spyder
+.. category: 
+.. link: 
+.. description: 
+.. type: text
+-->
+
+It has been almost two months since I joined Quansight, last April, to start 
+working with [Spyder](https://github.com/spyder-ide/spyder/) maintenance and 
+development. So far, it has been a very exciting and rewarding journey under 
+the guidance of long time Spyder maintainer
+[Carlos Córdoba](https://github.com/ccordoba12).
+This is the first of a series of blog posts we will be writing to showcase 
+updates on the development of Spyder, new planned features and news on the 
+road to Spyder 4.0 and beyond.
+
+Before we describe the features that will be included in the imminent release 
```

Comment:
Let's remove *imminent* from here because this is going to be published after beta2 is released.